maestro/02-launch-login/C311-relaunch-no-crash.yaml

```diff
@@ -0,0 +1,19 @@
+# Priority: Critical
+# Test ID: C311
+# Title: Verify you're able to Re-Launch/Open App
+# Expected Result: 
+#   1. Edge should not crash when relaunching the App
+
+appId: co.edgesecure.app
+tags:
+    - edge-
+---
+- runFlow:
+    file: ../common/launch.yaml
+
+- pressKey: home
+
+- runFlow:
+    file: ../common/launch.yaml
+
+    
```

Comment:
Does prettier or some other tool support YAML? Can we get an auto-formatter in here to ensure things like this don't happen?

Reply:
Yep! https://prettier.io/blog/2018/07/29/1.14.0.html

This needs to be tested, but I think it's a good starting point for our integration:

```json
{
  "scripts": {
    "fix": "eslint . --fix && yarn-deduplicate && prettier -w maestro/**/*.yaml"
    "lint": "eslint . && prettier maestro/**/*.yaml"
  },
  "lint-staged": {
    "*.{js,jsx,ts,tsx}": "eslint",
    "*.yaml": "prettier -w"
  }
}
```